addons/website_sale/static/src/js/location_selector/location/location.xml

```diff
@@ -1,14 +1,37 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="website_sale.locationSelector.location">
-        <div
-            class="border border-light"
-            t-att-class="{'o_location_selector_location_selected': props.isSelected}"
+        <button
+            class="collapsed list-group-item list-group-item-action d-flex gap-2 gap-lg-3"
+            t-att-class="{'o_location_selector_location_selected text-bg-light': props.isSelected}"
             t-on-click="() => props.setSelectedLocation(props.id)"
+            type="button"
```

Comment:
This may seem redundant, but it's always a good practice to specify the type of a button (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button), and it's also to maintain consistency with the other buttons implemented in Odoo :) 
![Capture d’écran 2024-04-29 à 12 28 32](https://github.com/odoo-dev/odoo/assets/80679690/96451b35-c317-4732-8551-89943b5903fb)